app/views/audits/common/_counter.html.erb

```diff
@@ -0,0 +1,3 @@
+<div class="content-items-counter">
+  <strong><%= format_number content_items.total_count %> items</strong>
```

Comment:
If you do `<% 'item'.pluralize(content_items.total_count) %>` it will read "1 item" not "1 items".